app/Foliage/Options.hs

```diff
@@ -157,3 +172,11 @@ importIndexCommand =
                 )
             )
       )
+
+toUppercase :: [Char] -> String
```

Comment:
There's nothing in `base` that already do the job?

Reply:
Crazy enough, no there is not :)